### PR TITLE
Catalog key v2

### PR DESCRIPTION
**Pull request by mrajashree**
https://github.com/rancher/rancher/issues/6390

### CODE REVIEW

File: manager/catalog.go [hunk 1]
```diff
@@ -292,17 +291,13 @@ func readRancherCompose(relativePath string, newTemplate *model.Template) error
 
 	//read the questions section
 	RC := make(map[string]model.RancherCompose)
```
Comment (joshwget) [inline manager/catalog.go]: Can you remove the `RC` variable and just use `catalogConfig`?

Comment (mrajashree) [inline manager/catalog.go]: done

**Comment by joshwget** [comment]
LGTM

**Comment by cjellick** [comment]
LGTM

**Pull request merged**